.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+
+      - name: Set up Go
+        uses: actions/setup-go@v2.1.2
+        with:
+          go-version: '1.15.2'
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2.3.2
+
+      - name: Run build
```

Comment:
Can we have `make ci` instead of those 5 separate targets? This way, it's easy to run checks locally before pushing.